website/docs/installing-podman-desktop/alternate-installation-methods.md

```diff
@@ -0,0 +1,235 @@
+---
+sidebar_position: 2
+title: Alternate installation methods.
```

Comment:
Nit: shouldn't end in a period, it's inconsistent and stands out in the nav.

Reply:
done